src/nvim/ex_docmd.c

```diff
@@ -721,15 +687,13 @@ int do_cmdline(char *cmdline, LineGetter fgetline, void *cookie, int flags)
     // Convert an interrupt to an exception if appropriate.
     (void)do_intthrow(&cstack);
   }
-  /*
-   * Continue executing command lines when:
-   * - no CTRL-C typed, no aborting error, no exception thrown or try
-   *   conditionals need to be checked for executing finally clauses or
-   *   catching an interrupt exception
-   * - didn't get an error message or lines are not typed
-   * - there is a command after '|', inside a :if, :while, :for or :try, or
-   *   looping for ":source" command or function call.
-   */
+  // Continue executing command lines when:
+  // - no CTRL-C typed, no aborting error, no exception thrown or try
+  //   conditionals need to be checked for executing finally clauses or
+  //   catching an interrupt exception
+  // - didn't get an error message or lines are not typed
+  // - there is a command after '|', inside a :if, :while, :for or :try, or
+  //   looping for ":source" command or function call.
   while (!((got_int || (did_emsg && force_abort) || current_exception)
            && cstack.cs_trylevel == 0)
          && !(did_emsg
```

Comment:
Change the `;` below to `{}`

Reply:
I tried that but it didn't work. It seemed like the expression was too complex for the linter or something.

Reply:
Oh, I didn't notice that this is a `do` ... `while`. Then I know how to fix this.